airbyte-webapp/package.json

```diff
@@ -7,11 +7,12 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "BUILD_PATH='./build/app' react-scripts build",
```

Comment:
:information_source: Move the actual react build output to `build/app` instead of `build` so we don't mix it with the docker and storybook output.